ros_ws/src/teleoperation/include/keyboard_controller.h

```diff
@@ -30,12 +32,21 @@ enum class KeyIndex : int
     DECELERATE = 2,
     STEER_LEFT = 1,
     STEER_RIGHT = 3,
-    DEAD_MANS_SWITCH = 4
+    ENABLE_MANUAL = 4,
+    ENABLE_AUTONOMOUS = 5
 };
 
-constexpr int KEY_COUNT = 5;
+constexpr int KEY_COUNT = 6;
 
-constexpr std::array<Keycode, KEY_COUNT> KEY_CODES = { Keycode::W, Keycode::A, Keycode::S, Keycode::D, Keycode::SPACE };
+constexpr std::array<Keycode, KEY_COUNT> KEY_CODES = { Keycode::W, Keycode::A,     Keycode::S,
+                                                       Keycode::D, Keycode::SPACE, Keycode::B };
+
+enum class DriveMode : int
+{
```

Comment:
Would it be possible to somehow move the `enum class DriveMode` to a common library for both the *teleoperation* and *car_control* program? If this is not possible in a reasonable manner, than this suggestion could be moved in a new issue (with low priority).

Reply:
We discussed the same issue when we realized that topic names are redefined in multiple packages. We decided to duplicate the names to avoid dependencies between the packages. I think this is a similar case. cc @StefanJVA

Reply:
True